python/poetry/Portfile

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+
+name                    poetry
+version                 1.0.3
+revision                0
+categories-append       devel
+platforms               darwin
+license                 MIT
+supported_archs         noarch
+
+maintainers             {gmail.com:davidgilman1 @dgilman} openmaintainer
+
+description             Python dependency management and packaging made easy.
+
+long_description        Poetry: Dependency Management for Python \
+                        \
+                        Poetry helps you declare, manage and install \
+                        dependencies of Python projects, ensuring you have \
+                        the right stack everywhere.
+
+homepage                https://python-poetry.org/
+
+checksums               rmd160  d6094162b8f01e15d86fb40b7aaab7e15bce76f1 \
+                        sha256  60a079b1c1d8a4f1538d28ee8d1bf79a5d0eae96f497fdae9512eb3c1f8da13b \
+                        size    164894
+
+python.default_version  38
+
+# These are in setup.py order
+depends_lib-append \
+    port:py${python.version}-cachecontrol \
+    port:py${python.version}-lockfile \
+    port:py${python.version}-cachy \
+    port:py${python.version}-cleo \
+    port:py${python.version}-clikit \
+    port:py${python.version}-html5lib \
+    port:py${python.version}-jsonschema \
+    port:py${python.version}-pexpect \
+    port:py${python.version}-pkginfo \
+    port:py${python.version}-parsing \
+    port:py${python.version}-pyrsistent \
+    port:py${python.version}-requests-toolbelt \
+    port:py${python.version}-requests \
+    port:py${python.version}-shellingham \
+    port:py${python.version}-tomlkit \
+    port:py${python.version}-keyring
+
```

Comment:
you seem to be missing a dependency on ```py-setuptools```, which is imported in ```setup.py```. In this case, since a ```console_script``` is installed it should be a ```depends_lib``` as it is also needed at runtime.